### ISSUE TITLE
[feature request] Accept a express instance as optional parameter in the feathersExpress.

### ISSUE THREAD

**Issue by TobyMosque** [feathersjs/feathers#1419]
Accept a express instance as optional parameter in the feathersExpress.

**Current**
```js
// link: https://github.com/feathersjs/feathers/blob/master/packages/express/lib/index.js
// imports
function feathersExpress (feathersApp) {
  if (!feathersApp) {
    return express();
  }

  if (typeof feathersApp.setup !== 'function') {
    throw new Error('@feathersjs/express requires a valid Feathers application instance');
  }

  if (!feathersApp.version || feathersApp.version < '3.0.0') {
    throw new Error(`@feathersjs/express requires an instance of a Feathers application version 3.x or later (got ${feathersApp.version || 'unknown'})`);
  }

  const expressApp = express();
  // do something
}
```
**Proposed**
```js
// imports
function feathersExpress (feathersApp, expressApp) {
  if (!feathersApp) {
    return expressApp || express();
  }

  if (typeof feathersApp.setup !== 'function') {
    throw new Error('@feathersjs/express requires a valid Feathers application instance');
  }

  if (!feathersApp.version || feathersApp.version < '3.0.0') {
    throw new Error(`@feathersjs/express requires an instance of a Feathers application version 3.x or later (got ${feathersApp.version || 'unknown'})`);
  }

  expressApp = expressApp || express();
  // do something
}
```

**Comment by MichaelJCole** [comment]
Hi @daffl this is part of the Quasar + Feathers conversation with @nothingismagick.  @TobyMosque is on the team and this change is toward integrating the dev experience.

**Comment by daffl** [comment]
Sounds good. I'll try and put up a PR for this today.

**Comment by daffl** [comment]
Released in v4.0.0-pre.5
